test-requirements.txt

```diff
@@ -24,7 +24,6 @@ pytest-reorder~=0.1.0
 git+https://github.com/jayvdb/pytest-reqs@coala#egg=pytest-reqs
 pytest-timeout~=1.3.0
 pytest-travis-fold~=1.3.0
-pytest-xdist~=1.15
```

Comment:
Remove `Closes #4061` from the commit summary -- that problem still exists -- we do want xdist to work -- there is a bug which needs to be solved.

We have been using `Related to <URL>` for this type of loose relationship to an issue.

Reply:
wtb `Asyncio.py`? Quotation issues is causing build failure :disappointed: